packages/synapse-react-client/src/components/SubmissionViewScopeEditor/SubmissionViewScopeEditor.tsx

```diff
@@ -0,0 +1,96 @@
+import React from 'react'
+import { ACCESS_TYPE } from '@sage-bionetworks/synapse-types'
+import EvaluationFinder from '../EvaluationFinder/EvaluationFinder'
+import { useGetEvaluation } from '../../synapse-queries/evaluation/useEvaluation'
+import { Box, IconButton, Skeleton, Typography } from '@mui/material'
+import IconSvg from '../IconSvg'
+
+type SubmissionViewScopeItemProps = {
+  evaluationId: string
+  onRemove: () => void
+}
+
+function SubmissionViewScopeItem(props: SubmissionViewScopeItemProps) {
+  const { evaluationId, onRemove } = props
+  const { data: evaluation, isLoading } = useGetEvaluation(evaluationId)
+  if (isLoading) {
+    return <Skeleton width={'100%'} />
+  }
+  return (
+    <Box
+      display={'flex'}
+      flexDirection={'row'}
+      gap={1}
+      alignItems={'center'}
+      justifyContent={'space-between'}
+    >
+      <Typography key={evaluation!.id} variant={'smallText1'}>
+        {evaluation!.name}
+      </Typography>
+      <IconButton
+        size={'small'}
+        aria-label={`Remove ${evaluation!.name!} from scope`}
+        onClick={() => {
+          onRemove()
+        }}
+        sx={{
+          '&:hover': {
+            color: 'error.main',
+          },
+        }}
+      >
+        <IconSvg icon={'delete'} fontSize={'inherit'} wrap={false} />
+      </IconButton>
+    </Box>
+  )
+}
+
+export type SubmissionViewScopeEditorProps = {
+  evaluationIds: string[]
+  onChange: (evaluationIds: string[]) => void
+}
+
+/**
+ * Component used to select evaluations to define a SubmissionView scope
+ * @param props
+ * @constructor
+ */
+export default function SubmissionViewScopeEditor(
```

Comment:
Slight improvement here where we show the selection at the same time as the full list of evaluations. This way you don't have to open a new modal to add each individual evaluation